features/exit_status.feature

```diff
@@ -1,16 +1,21 @@
-Feature: running tests has the correct exit status
+Feature: Signaling the test result through the command exit status
+
+  As an administrator integrating Mycha-based test runs into my automation pipeline
+  I want to capture the result of the test suite through the exit status of the application
+  So that I can use Mycha with my standard test toolset.
```

Comment:
Its a bit specific to one use case, but also nice, precise, and concise. Adopted.